addon/store.js

```diff
@@ -48,25 +51,32 @@ class PromiseTracker {
   }
 }
 
-var Store = Ember.Object.extend({
-  orbitStore: Ember.inject.service('orbitStore'),
+export default Ember.Object.extend({
+  orbitStore: null,
   schema: null,
   cache: null,
   _identityMap: null,
 
-  init: function(...args) {
-    this._super(...args);
-    const { orbitStore, container } = this.getProperties('orbitStore', 'container');
+  init() {
```

Comment:
The guarded injections seem like a bit of a smell here, perhaps we could move them out to an initializer?